src/OrbitGl/TrackManager.cpp

```diff
@@ -338,11 +338,13 @@ ThreadTrack* TrackManager::GetOrCreateThreadTrack(int32_t tid) {
 
 GpuTrack* TrackManager::GetOrCreateGpuTrack(uint64_t timeline_hash) {
   std::lock_guard<std::recursive_mutex> lock(mutex_);
-  std::shared_ptr<GpuTrack> track = gpu_tracks_[timeline_hash];
+  std::string timeline =
+      app_->GetStringManager()->Get(timeline_hash).value_or(std::to_string(timeline_hash));
```

Comment:
Why not use Track's name? If you don't like that there could be several with the same name (""), you could add the default value "timeline_hash" to Track's name in the constructor.

Reply:
I can't, as the track might have not yet been created. However, I also adjusted the Track's name.

Reply:
Yes, you are right. I only was worried about using twice this "GetTimeline". I've just checked if we can only give timeline instead of the hash to the GpuTrack constructor, but seems to be that we also need the timeline_hash for other functions (GetLeft and GetRight).